stable/prometheus-mongodb-exporter/templates/_helpers.tpl

```diff
@@ -51,4 +51,15 @@ Determine secret name, can either be the self-created of an existing one
 {{- else -}}
     {{ include "prometheus-mongodb-exporter.fullname" . }}
 {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{/*
+Determine secret key, can either be the self-created of an existing one
+*/}}
+{{- define "prometheus-mongodb-exporter.secretKey" -}}
```

Comment:
Don't think this function is necessary. Can simplify with a default in values.yaml and use `{{ .Values.existingSecret.key }}` in manifests